ucsb-cs156-s24/proj-happycows-s24-4pm-7
# mz - adding "suspend" button to users table for admins

## Overview
In this PR, I added an "suspend" button column on the users table for admins to be able to remove certain users from ALL commons if they are causing issues. Currently, the button serves as a placeholder and clicking it does not actually redirect/do anything.

## Screenshots
<img width="827" alt="Screen Shot 2024-05-27 at 5 09 36 PM" src="https://github.com/ucsb-cs156-s24/proj-happycows-s24-4pm-7/assets/86985941/01e4ba57-6e50-458a-9cc0-ce3b5d98dae6">



## Feedback Request 
Please give suggestions on the following:
- Should users that are admins not have "Suspend" buttons next to them? Would it ever be necessary to suspend an admin from a commons?
- I'm not sure if this is the most intuitive design, I'm open to other ideas about how it could be implemented better. 

## Future Possibilities 
Some possible points could be:
- This is the beginning/part of the "suspend", and we will need to implement the actual remove page for it to work.

## Validation (Optional)
1. Download this branch.
2. Run the project.
3. Go to "Users" under "Admin".
4. Check that for all users you can see a button that says "Suspend".

## Tests

## Linked Issues
Related to #8, does not close it. 
Addresses sub-issue F1 within [EPIC:Admin can suspend/restore Users](https://github.com/ucsb-cs156/proj-happycows/issues/124)

## Dokku Deployment
[https://happycows-mczimmermann-dev.dokku-07.cs.ucsb.edu](https://happycows-mczimmermann-dev.dokku-07.cs.ucsb.edu)
